src/transform.js

```diff
@@ -647,6 +645,25 @@ function untransformObject(schema, className, mongoObject, isNestedObject = fals
 
     var restObject = untransformACL(mongoObject);
     for (var key in mongoObject) {
+      const specialKeys = [
```

Comment:
If babel is anything like typescript that would just be converted to ``var specialKeys``

So ideally this should be declared outside of the function, or at the very least outside of the loop.